BlogTemplate/Pages/Drafts.cshtml.cs

```diff
@@ -24,7 +24,7 @@ public DraftsModel(BlogDataStore dataStore)
 
         public void OnGet()
         {
-            Posts = _dataStore.GetAllPosts();
+            Posts = _dataStore.GetAllPosts().Where(post => !post.IsPublic);

```

Comment:
I prefer the name PrivatePosts because that's really what's in there. It's not "just a list of posts" - it's specifically PrivatePosts (or DraftPosts, or NonPublicPosts).